src/ports.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <inttypes.h>
+
```

Comment:
nie potrzebna pusta linia

Reply:
Autoformatter mi rozdziela globalne include od lokalnych, ale mogę to zmienić.